src/views/UI.vue

```diff
@@ -2,7 +2,6 @@
   <section>
     <page-title />
     <section-title />
-    <hash />
```

Comment:
Jak nie ma już wywołania do import też trzeba wywalić, bo obciąża nam aplikacje.

Reply:
Usunięto import Hash oraz export default Hash